forge.config.ts

```
@@ -9,10 +10,34 @@ import { FuseV1Options, FuseVersion } from "@electron/fuses";
+    return { target: "dev", version: releaseRcMatch[1] };
+  }
+
+  const releaseMatch = branch.match(/^release\/([\d]+\.\d+\.\d+)$/);
+  if (releaseMatch && releaseRcMatch[1] === packageJSON) {
+    return { target: "prod", version: releaseMatch[1] };
+  }
+
+  return { target: "unknown" }; // fallback
+}
+
+const { target, version } = resolveTargetAndVersion(gitBranch);
```

ESLint

[error] 'version' is assigned a value but never used. Allowed unused vars must match /^__/u.

(@typescript-eslint/no-unused-vars)